native_client/stt.cc

```diff
@@ -167,6 +176,23 @@ StreamingState::processAudioWindow(const vector<float>& buf)
   pushMfccBuffer(mfcc);
 }
 
+void
+StreamingState::flushBuffer()
+{
+  // Flush audio buffer
+  processAudioWindow(audio_buffer_);
+
+  // Add empty mfcc vectors at end of sample
+  //for (int i = 0; i < model_->n_context_; ++i) {
+  //  addZeroMfccWindow();
+  //}
```

Comment:
Is this a comment or commented-out code? Why is this needed?

Reply:
Oops. Fixed